config/config-mysql_apache_jvm_statsd.yaml

```diff
@@ -1,6 +1,7 @@
 # This config mimicks the Cloud Monitoring Agent by collecting metrics from popular third-party services the Agent already supports
 # and formatting the metrics in the same way, so they appear similarly in Cloud Monitoring. The services targeted here are MySQL, Apache and JVM.
 # There is also support for DogStatsD metrics, with no transformations
+# Please configure your environment-specific details in the receivers part, especially for MySQL and JVM
 # Please remove/comment-out the pipelines you don't need/want in the last section.
 # Documentation specific to each exporter is found in the comment directly above its configuration
 
```

Comment:
Can you add a tab before line 22

Reply:
Nit: Also for the comments at the top of this file, it'd be better to wrap them at a regular line length rather than having some really long lines